### DISCUSSION TITLE
How to summerize across a row?

### DISCUSSION THREAD

**Discussion by jkroepke** [grafana/grafana-infinity-datasource#849] [Specific APIs] [answered]
I have a dynamic amount of colums and I would like to have an additional summerize colume.

<img width="1196" alt="image" src="https://github.com/grafana/grafana-infinity-datasource/assets/1560587/84b4ddab-0162-4002-9b1a-f55620a8590a">

How I can archive that?

**Comment by yesoreyeram** [comment]
What is your data(json/CSV) looks like? What parser you use? Should be possible with backend and jsonata

**Reply by jkroepke** [reply]
Azure Cost API, like this: https://github.com/grafana/grafana-infinity-datasource/blob/main/testdata/misc/azure-cost-management-daily.json

But only with x num of columns and x num of rows.

**Reply by jkroepke** [reply, marked as answer]
I was able to get this via transformation!
